src/arizona_live_view.erl

```diff
@@ -56,12 +56,8 @@ Live view.
 %% --------------------------------------------------------------------
 
 parse_str(Str, Macros) ->
-    case arizona_tpl_scan:string(Str) of
-        {ok, Tokens, _EndLocation} ->
-            arizona_tpl_parse:parse_exprs(Tokens, Macros);
-        {error, ErrorInfo, ErrorLocation} ->
-            {error, {ErrorInfo, ErrorLocation}}
```

Comment:
This cannot be returned at this moment, so it's dead code. Better wait until we actually implement it 😄